op-e2e/setup.go

```diff
@@ -85,7 +85,7 @@ func DefaultSystemConfig(t *testing.T) SystemConfig {
 		L1GenesisBlockBaseFeePerGas: uint642big(7),
 
 		L2GenesisBlockNonce:         0,
-		L2GenesisBlockGasLimit:      8_000_000,
+		L2GenesisBlockGasLimit:      30_000_000,
```

Comment:
Why does this change? Should it be 20M like in the system config?

Reply:
the invariant must hold that `deposit gas limit + system tx gas limit < L2 block gas limit`

Reply:
OK, I'm just trying to understand; it could be as low as 21 million, but we're just bumping it up to 30  because it's cleaner?

(please resolve if that's correct)

Reply:
Yup exactly! Figured we should just do what mainnet does